CHANGELOG.asciidoc

```diff
@@ -21,6 +21,22 @@ endif::[]
 [[release-notes-1.x]]
 === .NET Agent version 1.x
 
+[[release-notes-1.5.0]]
+==== 1.5.0
+
+[float]
+===== Features
+- Auto instrumentation for `SqlClient` (https://www.elastic.co/guide/en/apm/agent/dotnet/current/setup.html#setup-sqlclient[documentation])
+- Introducing Filter API {pull}792[#792] (https://www.elastic.co/guide/en/apm/agent/dotnet/current/public-api.html#filter-api[documentation])
+- Auto-detect culprit for exceptions {pull}740[#740]
+- New config settings: `ExcludedNamespaces`, `ApplicationNamespaces` (https://www.elastic.co/guide/en/apm/agent/dotnet/current/config-all-options-summary.html[documentation])
```

Comment:
```suggestion
- Auto instrumentation for `SqlClient` (<<setup-sqlclient,documentation>>)
- Introducing Filter API {pull}792[#792] (<<filter-api,documentation>>)
- Auto-detect culprit for exceptions {pull}740[#740]
- New config settings: `ExcludedNamespaces`, `ApplicationNamespaces` (<<config-all-options-summary,documentation>>)
```

Reply:
This should fix all of the documentation errors as all of this content exists in master. When the release is pushed to 1.x, all of the content and links in the 1.x branch will be updated to point to current instead.

The only downside of this is that the links are not clickable in markdown. I think that's ok though, as neither are the PR links, and we have a header at the top that encourages readers to view RNs in the documentaiton.